orchestration/templates/ingest-clinvar-archive.yaml

```diff
@@ -108,6 +108,20 @@ spec:
                   value: '{{ "{{inputs.parameters.gcs-prefix}}" }}/raw/VariationArchive'
                 - name: max-concurrent-uploads
                   value: '2'
+          # 5b) Extract clinvar release date.
+          - name: extract-release-date
+            dependencies: [extract-xml]
+            templateRef:
+              name: extract-from-json
+              template: main
+            arguments:
+              parameters:
+                - name: pvc-name
+                  value: '{{ $extractionPvc }}'
+                - name: file-path
+                  value: 'ClinVarVariationRelease/part-1.json'
```

Comment:
I wasn't sure about whether or not to include a forward slash at the beginning of this value

Reply:
We can code it either way, but I think it's fine to leave off the slash